### ISSUE TITLE
Create a database mockup

### ISSUE THREAD

**Issue by runjak** [runjak/TranscriptionDesk#5]
* Remember to tack timestamps on everything possibly useful
* Design with URNs in mind
    * SQL allows selection by prefixes, which might come in handy
* Create an ER diagram
* Produce working SQL to create tables

**Comment by runjak** [comment]
Please note that with 0c645685 there exists a method [```OmekaItem.getUrn()```](https://github.com/runjak/TranscriptionDesk/blob/master/src/omeka/item.php#L69-L80), which provides URNs given by Omeka that look like ```urn:cite:ogl:belluno_lol25```.

**Comment by runjak** [comment]
We created an ER diagram for our database today, which can be viewed at [notes/ER.2015-06-14.svg](https://github.com/runjak/TranscriptionDesk/blob/master/notes/ER.2015-06-14.svg).
I've also added [notes/2015-06-24.md](https://github.com/runjak/TranscriptionDesk/blob/master/notes/2015-06-24.md) which includes some notes on this.

**Comment by runjak** [comment]
I've added a draft [```create.sql```](https://github.com/runjak/TranscriptionDesk/blob/master/src/database/create.sql) to describe our database schema.

**Comment by runjak** [comment]
I've taken the liberty to update our docker setup so that a simple ```make build``` will give you a docker container that comes with MySQL fully configured to fit the defaults from ```config_example.php```.
If you're running a non docker setup, you'll need to execute ```create.sql``` against your database manually and adjust the login info in your ```config.php```. For more details see 6369a8972d115c30f70b143f0e1eb6348947c675.

**Comment by runjak** [comment]
I've figured out the max length of a ```VARCHAR``` that needs to be a key attribute in MySQL, after @ThomasK81 nudged me in that direction, and the database now can be created from ```create.sql```.
I've also made sure that our setup scripts for both docker and vagrant already fill the database from this so that it is ready to use. If we find that this schema is OK to work with for now, occasional adjustments aside, I would consider this issue solved. This is also something that we can discuss further on Friday.
